src/main/kotlin/org/eclipse/tractusx/managedidentitywallets/models/ssi/RegisterNymPublicDto.kt

```diff
@@ -0,0 +1,39 @@
+/********************************************************************************
```

Comment:
This file looks like a leftover, do we still need it?

Reply:
done